### DISCUSSION TITLE
[Proposal] Default partial flag

### DISCUSSION THREAD

**Discussion by workgroupengineering** [dotnet/csharplang#7932] [Language Ideas] [answered]
Roslyn's source generators have become very popular, but most project and item templates do not have a `partial` modifier in the class declaration, forcing you to modify each class definitions every time or generate custom templates. 

It would be handy if there was the possibility of setting a property in `csproj` that would make all classes without `partial` modifier considered partial.

**Comment by ufcpp** [comment, marked as answer]
https://github.com/dotnet/csharplang/issues/950
https://github.com/dotnet/csharplang/discussions/3748
